superset/connectors/sqla/models.py

```diff
@@ -916,7 +916,8 @@ def mutate_query_from_config(self, sql: str) -> str:
 
         Typically adds comments to the query with context"""
         sql_query_mutator = config["SQL_QUERY_MUTATOR"]
-        if sql_query_mutator:
+        mutate_after_split = config["MUTATE_AFTER_SPLIT"]
```

Comment:
New config vars should be added to config.py with a default value and a comment explaining what it does.

Reply:
config.py has been updated with a default value and comment on usage